operating_unit/__manifest__.py

```diff
@@ -12,7 +12,7 @@
     "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/operating-unit",
     "category": "Generic",
-    "depends": ["base"],
+    "depends": ["base", "base_setup"],
```

Comment:
Please, remove this dependency. It may generate problems in databases that already have this module installed. In fact, this dependency is not needed: those who have "base_setup" installed, will get the bonus from the `default_get`. If they don't have it installed, then the `default_get` will do nothing.